src/sundials/sundials_context.c

```diff
@@ -199,8 +259,7 @@ int SUNContext_Free(SUNContext* sunctx)
 
   /* Enforce that the profiler is freed before finalizing,
      if it is not owned by the sunctx. */
-  if ((*sunctx)->profiler)
-  {
+  if ((*sunctx)->profiler) {
     if (fp) SUNProfiler_Print((*sunctx)->profiler, fp);
     if (fp) fclose(fp);
     if ((*sunctx)->own_profiler) SUNProfiler_Free(&(*sunctx)->profiler);
```

Comment:
I think it would be a bit strange and possibly lead to errors if we use it here. If code gets moved around and the logger is freed first, it would not be able to log, for example.